packages/flutter_tools/templates/create/android.tmpl/buildSrc/src/main/groovy/FlutterPlugin.groovy

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
Developers aren't supposed to edit this file.  We should probably say that explicitly and have the build check that it hasn't been modified.  (It's in a directory that developers normally don't expect to modify, but we should make that explicit.)